### PR TITLE
update demo with markdown

### PR DESCRIPTION
**Pull request by quincyle**
#major#
update demo with markdown
change to onInput action interface

### CODE REVIEW

**Comment by sandersky** [comment]
:-1: When changing the API it is a #breaking# change.

**Comment by rox163** [comment]
:+1:

File: tests/dummy/app/pods/demo/template.hbs [hunk 1]
```diff
@@ -1,64 +1,133 @@
...
-      on-input=(action "bind2")
-      value=boundText
-    }}
-  </div>
+        <div class="example">
+            <div class="title">Bound</div>
+            <div class="snippet">
+              {{format-markdown "```handlebars
+{{frost-text
+  onInput=(action 'bind')
```
Comment (sglanzer) [inline tests/dummy/app/pods/demo/template.hbs]: I think to make these examples clearer you should use the same action name in each (e.g. 'handleInputEvent')

**Comment by sglanzer** [comment]
Approved

**Pull request merged**